avatars/processors/proportions_test.py

```diff
@@ -155,4 +155,53 @@ def test_preprocess_target_variables_sum_zero() -> None:
         "variable_3": [0.9, 0],
     }
     )
-    pd_testing.assert_frame_equal(preprocessed, expected)
\ No newline at end of file
+    pd_testing.assert_frame_equal(preprocessed, expected)
+
+
+def test_postprocess_saferounding() -> None:
+    "Verify that the postprocessed variables sum up to reference variable." 
+    df = pd.DataFrame(
+        {
+            "variable_1": [101, 101],
+            "variable_2": [34, 20],
+            "variable_3": [66, 80],
+        }
+    )
+    processor = ProportionProcessor(
+        variable_names=["variable_2", "variable_3"],
+        reference="variable_1",
+        sum_to_one=True,
+        decimal_count=0
+    )
+    preprocessed = processor.preprocess(df=df)
```

Comment:
nit: if you test the postprocessed I would avoid the use of `preprocess`. You can take as parameter df and preprocessed